DirectX11Proj/texture.vs

```diff
@@ -0,0 +1,47 @@
+Texture2D shaderTexture;
+SamplerState SampleType;
+
+cbuffer MatrixBuffer
+{
+	matrix worldMatrix;
+	matrix viewMatrix;
+	matrix projectionMatrix;
+};
+
+struct VertexInputType
+{
+    float4 position : POSITION;
+    float2 uv : TEXCOORD0;
+	float3 normal : NORMAL;
```

Comment:
starting to believe there might be some parcing issues with github, not seeing this in vidusal studio